Dockerfile

```diff
@@ -13,6 +13,8 @@ RUN composer self-update --no-progress && \
 # Copy the working dir to the image's web root
 COPY . /var/www/html
 
+RUN chmod a+rwx /var/www/html/assets
```

Comment:
What was the error? You should never have to write to this directory. The correct assets directory is `/var/www/html/web/assets`. And that directory should already have the correct permissions.